Error: Unable to get local issuer certificate

Hi,

I've configured the vault-sync.yaml file as follows (just leaving the fields with an actual value set) :

```yaml

id: vault-sync
full_sync_interval: 3600 # 1h

# Source Vault configuration to sync secrets from.
src:
  # Vault URL
  url: http://source.vault:8200/

  # Vault Token auth method
  token: xxx.yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy

# Destination Vault configuration to sync secrets to.
dst:
  # Vault URL
  url: https://target.vault:8200/

  # Vault Token auth method
  token:xxx.yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy
```
I then try to run the install with :
`docker run -it -v $PWD:/vault-sync pbchekin/vault-sync:0.8.0 vault-sync --config /vault-sync/vault-sync.yaml`

Giving the output :
```
15:42:14 [INFO] Configuration from /vault-sync/vault-sync.yaml:
{
  "id": "vault-sync",
  "full_sync_interval": 3600,
  "bind": null,
  "src": {
    "url": "http://source.vault:8200/",
    "token": "***",
    "token_ttl": null,
    "token_max_ttl": null,
    "prefix": "",
    "backend": "secret"
  },
  "dst": {
    "url": "https://target.vault:8200/",
    "token": "***",
    "token_ttl": null,
    "token_max_ttl": null,
    "prefix": "",
    "backend": "secret"
  }
}
15:42:14 [INFO] Connecting to http://source.vault:8200/
15:42:14 [INFO] Audit device vault-sync exists: false
15:42:14 [INFO] Connecting to https://target.vault:8200/
15:42:14 [INFO] Token: TokenInfo { renewable: false, ttl: None, max_ttl: None }
15:42:14 [INFO] Plan: TokenInfo { renewable: false, ttl: None, max_ttl: None }
15:42:14 [ERROR] Failed to connect to https://target.vault:8200/: reqwest error: error sending request for url (https://target.vault:8200/v1/auth/token/lookup-self): error trying to connect: error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed:../ssl/statem/statem_clnt.c:1916: (unable to get local issuer certificate)
Error: Reqwest(reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("target.vault")), port: Some(8200), path: "/v1/auth/token/lookup-self", query: None, fragment: None }, source: hyper::Error(Connect, Ssl(Error { code: ErrorCode(1), cause: Some(Ssl(ErrorStack([Error { code: 337047686, library: "SSL routines", function: "tls_process_server_certificate", reason: "certificate verify failed", file: "../ssl/statem/statem_clnt.c", line: 1916 }]))) }, X509VerifyResult { code: 20, error: "unable to get local issuer certificate" })) })
```

The connection to the source Vault which is using HTTP doesn't have any issue (it would seem). However, the connection to the target Vault which is restricted to HTTPS only fails because the "local issuer certificate" cannot be accessed.

How could I provide the "local issuer certificate" when installing ?

P.S: I've tested connecting directly on the target Vault UI with the token I provided in vault-sync.yaml and was able to login.

Thank you.

Here is the logs:
```
07:08:48 [INFO] Connecting to https://vault.src/
07:08:48 [ERROR] Failed to connect to https://vault.src/: reqwest error: error sending request for url (https://vault.src/v1/auth/token/lookup-self): error trying to connect: error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed:../ssl/statem/statem_clnt.c:1916: (unable to get local issuer certificate)
Error: Reqwest(reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("vault.src")), port: None, path: "/v1/auth/token/lookup-self", query: None, fragment: None }, source: hyper::Error(Connect, Ssl(Error { code: ErrorCode(1), cause: Some(Ssl(ErrorStack([Error { code: 337047686, library: "SSL routines", function: "tls_process_server_certificate", reason: "certificate verify failed", file: "../ssl/statem/statem_clnt.c", line: 1916 }]))) }, X509VerifyResult { code: 20, error: "unable to get local issuer certificate" })) })
```
here is my values:
`
vault-sync:
  vaultSync:
    src:
      url: https://vault.src/
      backend: engine
      token: xxx.xxxxxxxxxxxxxxxxxxxxxx
    dst:
      url: https://vault.dst/
      backend: engine
      token: xxx.xxxxxxxxxxxxxxxxxxxxxx
`

--- a/README.md
+++ b/README.md
@@ -187,3 +187,32 @@ helm search repo vault-sync
 # create myvalues.yaml, using install/helm/values.yaml as the example
 helm install vault-sync vault-sync/vault-sync -f myvalues.yaml
 ```
+
+### Custom CA certificates
+
+When running vault-sync locally, specify environment variable `SSL_CERT_FILE` pointing to a PEM file with a certificate or certificates (see https://docs.openssl.org/3.1/man7/openssl-env/).
+
+When running vault-sync in Kubernetes, first create a Kubernetes secret from the PRM file.
+For example:
+
+```shell
+kubectl --namespace=vault-sync create secret generic certs --from-file=ca-bundle.pem=/path/to/certificate.pem
+```
+
+Then specify the following Helm chart values:
+
+```yaml
+volumes:
+  - name: certs
+    secret:
+      secretName: certs
+
+volumeMounts:
+  - mountPath: /certs
+    name: certs
+    readOnly: true
+
+environmentVars:
+  - name: SSL_CERT_FILE
+    value: /certs/ca-bundle.pem
+```

